src/gui/redux/docgen/docgen.js

```diff
@@ -0,0 +1,24 @@
+import {
+  REQUEST_DOCGEN,
+  RECEIVED_DOCGEN,
+  RECEIVED_EMPTY_DOCGEN,
+  FAILED_DOCGEN,
+} from './docgen.actions'
+
+export const EMPTY_DOCGEN = {}
+export const LOADING_DOCGEN = {}
```

Comment:
This kind of use of instance is meh for me.
If you look at state (via `redux-devtools`), you don't know if the state is in an empty state, or a loading state.

This is an unnecessary hack to me.
Why don't we use booleans ?